plugins/azad-proxy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: azad-proxy
```

Comment:
If the plugin remains focused on handling authn via Azure AD, maybe call it `azad-auth` instead?

Reply:
I wrote a more detailed explination here regarding this: https://github.com/kubernetes-sigs/krew-index/pull/997#issuecomment-766459068

Do you still want me to change it? I would rather not since I do think this is the name it should be to match the project, but I will of course follow your recommendations if you still think it's neccesary.

Thanks!